drivers/espi/espi_mchp_xec_host_v2.c

```diff
@@ -754,6 +881,10 @@ static const struct espi_lpc_req espi_lpc_req_tbl[] = {
 #ifdef CONFIG_ESPI_PERIPHERAL_HOST_IO
 	{ EACPI_START_OPCODE, EACPI_MAX_OPCODE, eacpi_rd_req, eacpi_wr_req },
 #endif
+#if defined(CONFIG_ESPI_PERIPHERAL_EC_HOST_CMD) && \
+	defined(CONFIG_ESPI_PERIPHERAL_ACPI_SHM_REGION)
+	{ EACPI_GET_SHARED_MEMORY, EACPI_GET_SHARED_MEMORY, eacpi_shm_rd_req, eacpi_shm_wr_req},
```

Comment:
nit: spaces on inside of { and } to be consistent ... you only have it on one side here